### ISSUE TITLE
No new data since november

### ISSUE THREAD

**Issue by razzeee** [Arxcis/flatstat#30]
It seems like there was no new data since november?

You might want to use the api of the website https://flathub.org/api/v2/docs for querying summaries as that also includes firefox etc that are not on the github.

But can't give any stability gurantees on the above api.

**Comment by Arxcis** [comment]
New release out now https://github.com/Arxcis/flatstat/releases/tag/v2025.07 

Sorry for infrequent updates 🙏 

Thank you for the tip on Flathub API. I was not aware of this until you mentioned it. I will look into it. I am not sure it will give me the same historical data of changes to "finish args" though... 🤔

**Comment by razzeee** [comment]
> New release out now [`v2025.07` (release)](https://github.com/Arxcis/flatstat/releases/tag/v2025.07)
> 
> Sorry for infrequent updates 🙏
> 
> Thank you for the tip on Flathub API. I was not aware of this until you mentioned it. I will look into it. I am not sure it will give me the same historical data of changes to "finish args" though... 🤔

You're correct, changes to `finish args` are not kept historically (I wanted to do that, but couldn't figure out something needed for it)

If you end up using the api, please don't use the compat endpoints :)
